interfaces/IBrowser.h

```diff
@@ -149,6 +161,14 @@ namespace Exchange {
 
         // @brief Initiate garbage collection
         virtual uint32_t CollectGarbage() = 0;
+        // @property
+        // @brief Languages: Browser prefered languages
```

Comment:
Don't prepend brief with the method name, the generator already knows it.

Reply:
done